src/utils/constants/apis.js

```diff
@@ -0,0 +1,14 @@
+export const API_TO_GET_ALL_CATEGORIES = {
+  api: "/api/categories",
+  propertyToGet: "categories",
+};
+
+export const API_TO_GET_ALL_PRODUCTS = {
+  api: "/api/products",
+  propertyToGet: "products",
+};
+
+export const API_TO_GET_HOME_PRODUCTS = {
+  api: "/api/products/home",
+  propertyToGet: "products",
+};
```

Comment:
Appreciation - this is very nice, you re maintaining an object for API calls & based on that calling a single hook. Even I will be using this.